packages/tools/u-boot/package.mk

```diff
@@ -29,8 +32,8 @@ case "$PROJECT" in
     PKG_PATCH_DIRS="rockchip"
     ;;
   *)
-    PKG_VERSION="v2020.10"
-    PKG_SHA256="0c022ca6796aa8c0689faae8b515eb62ac84519c31de3153257a9ee0f446618f"
+    PKG_VERSION="v2021.01-rc4"
```

Comment:
are you okay with that change ? @jernejsk @Kwiboo @chewitt

Reply:
I'm using the same so this is no drama for me. It should go final release mid-Jan.

Reply:
I'm fine with that